core-cube/src/main/java/org/apache/kylin/cube/model/validation/rule/DictionaryRule.java

```diff
@@ -89,17 +100,32 @@ public void validate(CubeDesc cubeDesc, ValidateContext context) {
                 return;
             }
 
+            if (StringUtils.isNotEmpty(builderClass) && !builderClass.equalsIgnoreCase(GlobalDictionaryBuilder.class.getName()) && bitmapMeasures.containsKey(dictCol) && !dictCol.getColumnDesc().getType().isIntegerFamily()){
```

Comment:
"builderClass" can be empty, which would means the normal dictionary. So this case should also be checked.